product-service/serverless.yml

```diff
@@ -13,6 +13,12 @@ provider:
 
   environment:
     AWS_NODEJS_CONNECTION_REUSE_ENABLED: 1
+    PG_HOST: lesson-4.cxhfwgzvdjjg.eu-west-1.rds.amazonaws.com
+    PG_PORT: 5432
+    PG_DATABASE: products_service
+    PG_USERNAME: postgres
+    PG_PASSWORD: qwerty123
```

Comment:
Just a note - it is not safe to publish DB password to VCS

Reply:
My bad, it was a late night commit. But qwerty123 is not a real password )
Nevertheless your're right, I should'nt have put it in the commit.